react-native-in-app-payments-quickstart/app/components/CommonAlert.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ Copyright 2022 Square Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+import React from 'react';
+import { Text, StyleSheet, Image, View, Modal, TouchableOpacity } from 'react-native';
+import PropTypes from 'prop-types';
+
+const FailIcon = require('../images/failIcon.png');
+const SuccessIcon = require('../images/sucessIcon.png');
+
+AddressView.propTypes = {
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+};
+
+export default function AddressView({ title, description, status, isVisible, onDialogClick }: { title: string, description: string, status: boolean, isVisible: boolean, onDialogClick: Function }) {
```

Comment:
is AddressView the right name for this component? The file name is CommonSlert.tsx

Reply:
Done